layouts/lists.tsx

```diff
@@ -36,21 +39,13 @@ const ListPageLayout = ({ children }: { children: React.ReactNode }) => {
         <div className="info-container container w-full min-h-[100px]">
           <Header>
             {list ? (
-              <ListHeader list={list} listId={list.id} isOwner={isOwner} />
-            ) : isLoading ? (
-              <div className="flex justify-between w-full h-46">
```

Comment:
The skeleton wrapper is no longer required as this renders server side via `getServerSideProps`.

Reply:
This layout has been reworked to receive props now.

Reply:
Note: This is a change in how we've done layouts. Layouts are applied here https://github.com/open-sauced/app/blob/beta/pages/_app.tsx#L160, which is why they have only `children` as props.

So each page under `pages/lists` would need to use the layout the same way

Reply:
Yeah, I saw that. The overview page doesn't really have anything at the moment and the Activity page does not exist yet.

I made this change so that a layout could benefit from SSR generated props. I'd apply the same pattern to those other pages for lists, or do you think it would cause issues elsewhere? I believe that template is only use on the overview and contributors list atm.

Reply:
Yep, this is a new layout/section, so I don't think it will cause issues elsewhere. I'll adjust my overview page after this lands.